src/app.scss

```diff
@@ -0,0 +1,348 @@
+@import url("https://fonts.googleapis.com/css2?family=Koulen&family=Source+Code+Pro&display=swap");
```

Comment:
Can we pull this font into local files and not use a web font? Otherwise if the phone is offline this won't load. Maybe there is a webpack plugin that does this automatically?